pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -45,7 +67,7 @@ while [ "$#" -gt 0 ]; do
         action="$i"
         ;;
       --install-grub)
-        echo "$0: --install-grub deprecated, use --install-bootloader instead" >&2
+        msg "$0: --install-grub deprecated, use --install-bootloader instead"
```

Comment:
This changes the semantics, since the message is no longer sent to stderr. You could either keep the redirect here, or use `$1` in `msg` to indicate the target file descriptor, as in `printf '%s\n' "$@" >&"$1"`.